### DISCUSSION TITLE
Cannot detect MulanPSL-2.0 license

### DISCUSSION THREAD

**Discussion by hktkzyx** [community/community#18206] [Projects and Issues]
According to the [GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license), MulanPSL-2.0 license is included in the *Choose a license* website and can be detected by the Licensee as shown in the [issue](https://github.com/licensee/licensee/issues/529). However, my repositories using MulanPSL-2.0 cannot be detected, e.g., https://github.com/hktkzyx/travel-map.

**Comment by DavidYang2149** [comment]
Hi~
I check GitHub docs, but there is not included MulanPSL-2.0 license in website.
Can you check again?

**Reply by hktkzyx** [reply]
I know MulanPSL-2.0 is not included in the section [Searching GitHub by license type](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#searching-github-by-license-type). However, according to section [Detecting a license](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license), MulanPSL-2.0 should be deteced.
